src/stores/configStore.js

```diff
@@ -187,8 +187,9 @@ export class ConfigStore {
           this[key] = getProtein('nsp12 - RdRp');
           this.urlParams.set(key, 'nsp12 - RdRp');
         }
-      } else if (key === 'ageRange') {
+      } else if (key === 'ageRange' || key === 'selectedPrimers') {
         // AgeRange is not being used currently so ignore
+        // selectedPrimers should not be set from the url
```

Comment:
Handle primers (https://github.com/vector-engineering/covidcg/blob/master/static_data/primers.csv)

Primer names are not unique but the pair of Institution + Name is unique, so we'll encode that and search by that using `queryPrimers` as defined in `utils/primer.js`

Example call:

```
queryPrimers({ Institution: "CDC", Name: "N1" })
```